src/main/java/md/miller1995/Dealership/controllers/CarController.java

```diff
@@ -22,11 +21,19 @@ public CarController(CarService carService){
         this.carService = carService;
     }
 
-    @GetMapping("/{id}")
+    @GetMapping("/search/by/{id}")
     public ResponseEntity<CarDTO> getCarById(@PathVariable(name = "id") long id){
         CarEntity carEntity = carService.findById(id);
         CarDTO carDTO = carService.carEntityToCarDTO(carEntity);
 
         return new ResponseEntity<CarDTO>(carDTO, HttpStatus.OK);
     }
+
+    @GetMapping("/search/by")
+    public ResponseEntity<List<CarDTO>> findCarByModel(@RequestParam(value = "model") String model){
```

Comment:
Even if we have a single property, we should use a DTO for it, not a String.

Reply:
@AlexandruIliescu 
- I don't understand how I need to do this, can you help me, please, with a hint  ?
- In my repository, method that do query for select all car by model from database has a parameter (String model).
- But in carController and carService I need to use CarDTO like as parameter in method.
- I try to obtain in carService -> with getter car.getModel and assignee this variable at String and after use this String into repository like a parameter, but it doesn't work.

Reply:
@Miller1995, my bad. I confused @RequestParam with @RequestBody on that line. 
The mapping for that API it's fine, you can make it a little bit simpler maybe by passing the value for the `@RequestParam` without using `value` notation:

```
@GetMapping("/search/by")
public ResponseEntity<List<CarDTO>> findCarByModel(@RequestParam( "model") String model){}

```

Please keep in mind that on URLs we should not use verbs.

Reply:
Thanks a lot!
- I changed the URL and I didn't use verbs, plus I used swagger and annotation @Operation for explain the APIs intention.

Reply:
Awesome, great job!